test/e2e/toolchain_test.go

```diff
@@ -36,19 +36,18 @@ func TestToolchain(t *testing.T) {
 	tektonSub := tekton.NewSubscription(tekton.SubscriptionNamespace)
 
 	cheInstallation := NewCheInstallation(cheOperatorNs)
+	tektonInstallation := NewTektonInstallation()
+
```

Comment:
I would prefer two test method `TestCheInstallation()` and `TestTektonInstallation()` instead of one currently `TestToolchain()` .. so adding more test in future will have some segregation.

Reply:
I believe, we are testing end to end, I don't want to seperate it, as we are testing anything randomly and with this way we'll be more confident about it's behaviour